template/package.json

```diff
@@ -157,6 +161,8 @@
     "preCommit": "node scripts/pre-commit.js",
     "postCss:component": "node ./scripts/postCss.js",
     "postinstall": "node packageScripts/postinstall.js",
+    "prettier:check": "npx prettier --check .",
+    "prettier:write": "npx prettier --write .",
```

Comment:
I feel like these should be installed directly. Set up with npx that means that this will be making an install call every time it runs.